GitCommands/Git/Tag/GitTagController.cs

```diff
@@ -9,40 +11,64 @@ public interface IGitTagController
         /// <summary>
         /// Create the Tag depending on input parameter.
         /// </summary>
-        /// <returns><see langword="true"/> if create tag command succeeded, <see langword="false"/> otherwise</returns>
-        GitCreateTagCmd GetCreateTagCommand(GitCreateTagArgs args);
+        /// <param name="args">tag creation arguments</param>
+        /// <param name="parentWindow">the UI window to act as the parent of the create tag dialog</param>
+        /// <returns>the true if the tag is created.</returns>
+        bool CreateTag(GitCreateTagArgs args, IWin32Window parentWindow);
     }
 
     public class GitTagController : IGitTagController
     {
-        private readonly Func<string> _getWorkingDir;
         private readonly IFileSystem _fileSystem;
+        private readonly IGitUICommands _uiCommands;
 
-        public GitTagController(Func<string> getWorkingDir, IFileSystem fileSystem)
+        public GitTagController(IGitUICommands uiCommands, IFileSystem fileSystem)
         {
-            _getWorkingDir = getWorkingDir;
             _fileSystem = fileSystem;
+            _uiCommands = uiCommands;
         }
 
-        public GitTagController(Func<string> getWorkingDir)
-            : this(getWorkingDir, new FileSystem())
+        public GitTagController(IGitUICommands uiCommands)
+            : this(uiCommands, new FileSystem() )
         { }
 
         /// <summary>
         /// Create the Tag depending on input parameter.
         /// </summary>
-        /// <returns>Output string from RunGitCmd.</returns>
-        public GitCreateTagCmd GetCreateTagCommand(GitCreateTagArgs args)
+        /// <param name="args">tag creation arguments</param>
+        /// <param name="parentWindow">the UI window to act as the parent of the create tag dialog</param>
+        /// <returns>the true if the tag is created.</returns>
+        public bool CreateTag(GitCreateTagArgs args, IWin32Window parentWindow)
         {
+            if(parentWindow == null)
```

Comment:
A few white-space violations in this diff. I expect the SCA stuff to fail the build.

Reply:
We don't have SLA in 2.x  :)

Reply:
I didn't notice which branch this PR was for...